### PR TITLE
Submission system - content edits

### PR DESCRIPTION
**Pull request by elizabethdryad**
Edited messages.xml file and a few java files to incorporate content edits from https://docs.google.com/a/datadryad.org/document/d/1cyBCUBRihCKtVAqFNNMKgQS81qbrInoyJOIuM7gG-W0/edit. Verified on localhost.

### CODE REVIEW

File: dspace/modules/payment-system/payment-api/src/main/java/org/dspace/paymentsystem/PaymentSystemImpl.java [hunk 1]
```diff
@@ -675,7 +675,7 @@ private void generateCountryList(org.dspace.app.xmlui.wing.element.List info,Pay
 
         if(shoppingCart.getCountry().length()>0)
         {
-            info.addItem("remove-country","remove-country").addXref("#","Remove Country : "+shoppingCart.getCountry());
+            info.addItem("remove-country","remove-country").addXref("#","Remove country : "+shoppingCart.getCountry());
```
Comment (ghost) [inline dspace/modules/payment-system/payment-api/src/main/java/org/dspace/paymentsystem/PaymentSystemImpl.java:678]: This and 'Remove Voucher' also need to be updated in the file `dryad-repo/dspace/modules/xmlui/src/main/webapp/themes/Mirage/lib/js/dryad-pages.js`

File: dspace/modules/xmlui/src/main/webapp/i18n/messages.xml [hunk 2]
```diff
@@ -540,8 +540,8 @@
     <message key="xmlui.EPerson.EditProfile.terms_help">To use Dryad's data publishing services, you must agree to the <a target="_blank" href="/themes/Mirage/docs/TermsOfService-Letter-2013.08.22.pdf">Terms of Service</a>.</message>
     <message key="xmlui.EPerson.EditProfile.terms.checkbox">I agree to Dryad's Terms of Service.</message>
 
-    <message key="xmlui.EPerson.EditProfile.submit_update">Complete Registration</message>
-	<message key="xmlui.EPerson.EditProfile.submit_create">Update Profile</message>
+    <message key="xmlui.EPerson.EditProfile.submit_update">Complete tegistration</message>
```
Comment (ghost) [inline dspace/modules/xmlui/src/main/webapp/i18n/messages.xml]: this button has the text `Complete tegistration`

**Comment by elizabethdryad** [comment]
Thanks for catching this stuff -- should be fixed

**Comment by ghost** [comment]
#### Code Review Checklist
- [x] Read through the code to ensure it is:
- [x] Legible
- [ ] Well-commented
- [ ] Free of any "code smells"
- [ ] Ensure the associated documentation is up to date.
- [x] Compile it on your local machine and test the new functionality.
- [ ] Verify that the change is accompanied by adequate testing code. Verify that the code passes the tests.

Updates test out in Travis and on local build, and updates display as expected in local app.

**Pull request merged**